### ISSUE TITLE
The problem of dividing Train set and Validation set

### ISSUE THREAD

**Issue by duyuwen-duen** [tum-traffic-dataset/coopdet3d#6]
I find that the training set and the validation set are almost identical, which means that a high mean Average Precision (mAP) does not necessarily indicate better detection ability; it could be overfitting. 
![image](https://github.com/user-attachments/assets/6d1723ca-763b-48fc-bc6a-dfd1ad8126a0)

I have separated the training set and the validation set, using the earlier data as the training set and the remaining data as the validation set. This means the validation set is distinct from the training set, and I have retrained the model. However, I found that the mAP has decreased significantly. This makes me question whether the original split method was reasonable. Looking forward to your reply!
![image](https://github.com/user-attachments/assets/e1e600d1-6241-4fef-8afe-fcda865e8822)
